Unable to load WebView2Loader.dll

### Description:
When installing the WinUI.TableView nuget-package and building my project I get the following build error:

```
..\Microsoft.Common.CurrentVersion.targets(5394,5): error MSB3030: Could not copy the file "C:\Users\______\.nuget\packages\winui.tableview\1.3.1\lib\net9.0-windows10.0.19041\WinUI.TableView\runtimes\win-arm64\native\WebView2Loader.dll" because it was not found.
..\Microsoft.Common.CurrentVersion.targets(5394,5): error MSB3030: Could not copy the file "C:\Users\______\.nuget\packages\winui.tableview\1.3.1\lib\net9.0-windows10.0.19041\WinUI.TableView\runtimes\win-x64\native\WebView2Loader.dll" because it was not found.
..\Microsoft.Common.CurrentVersion.targets(5394,5): error MSB3030: Could not copy the file "C:\Users\______\.nuget\packages\winui.tableview\1.3.1\lib\net9.0-windows10.0.19041\WinUI.TableView\runtimes\win-x86\native\WebView2Loader.dll" because it was not found.
```

![Image](https://github.com/user-attachments/assets/6ff479b5-0fce-4e9c-9b6a-e28e172a3934)

Is it related to one of these issues perhaps?

https://github.com/microsoft/WindowsAppSDK/issues/5058
https://github.com/microsoft/WindowsAppSDK/issues/4807

The nuget-package is installed in a class library, and I tried the solution here but without success: https://github.com/microsoft/WindowsAppSDK/issues/4807#issuecomment-2567255519

Had to copy the dll:s manually to the WinUI.TableView folder to be able to continue.

--- a/src/WinUI.TableView.csproj
+++ b/src/WinUI.TableView.csproj
@@ -27,6 +27,7 @@
   
   <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.250108002" />
+    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2651.64" PrivateAssets="all" />
   </ItemGroup>
   
   <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'windows'">

--- a/src/WinUI.TableView.csproj
+++ b/src/WinUI.TableView.csproj
@@ -27,7 +27,7 @@
   
   <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) == 'windows'">
     <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.6.250108002" />
-    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2651.64" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.2903.40" PrivateAssets="all" />
   </ItemGroup>
   
   <ItemGroup Condition="$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'windows'">

